src/sass/_config.scss

```diff
@@ -46,6 +46,84 @@ $transparent: rgba(255, 255, 255, 0);
 $buttonPrimaryFbColor: #3c5b96;
 $buttonPrimaryFbHoverColor: #4367a9;
 
+// New colors system ⬇️
+$blue-70: #002238;
```

Comment:
@domidomi this fake comment to mitigate "no usage" issue will be needed here, right?

Reply:
@coderitual Do you mean I should move it right below old color system?

Reply:
@domidomi oh sorry, I thought it was that error we have been takimg about but apparently there are new scams linter errrors:
- small hex values
- Camel case in variables - can we change this settting to allow the new name with hyphen?

Reply:
@coderitual it seems we can only change the rule `variable-name-format` to some regex expression, as it doesn't support multiple values. In that case I have to do changes to `sass-lint-config-brainly` package

Reply:
@domidomi let's override those rules in.sass-lint for now. Frontend app uses different linter and we need to check and eventually override rules in the app during sg upgrade.